src/js/modules/setup/services/new-account-form/new-account-form.service.js

```diff
@@ -38,20 +39,24 @@
             version: self._CONFIG.VERSION || self._CONFIG.VERSION_REV,
             device_uuid: self._device.uuid,
             device_name: (self._device.platform || self._device.model) ? ([self._device.platform, self._device.model].clean().join(" / ")) : "Unknown Device",
-            super_secret: self._CONFIG.SUPER_SECRET || null,
+            super_secret: null,
             powtcha: null,
             browser_fingerprint: null,
             skip_two_factor: true // will make the resulting API key not require 2FA in the future
         };
 
         var url = self._CONFIG.API_URL + "/v1/" + data.networkType + "/mywallet/register";
 
+        self._$log.debug("M:SETUP:newAccountFormService: register", postData.email, postData.platform, postData.device_name);
+
         return self._$http.post(url, postData)
-            .then(self._trackEvent.bind(self))
-            .then(self._storeAccountInfo.bind(self))
+            // .then(self._trackEvent.bind(self))
```

Comment:
bring back tracking events, they're very important for Nikols conversion tracking

Reply:
Return and create a ticket to check all track events trough the app

https://trello.com/c/6ICnNdpz/16-check-tracking-events-through-the-app-trackingservicetrackevent